### PR TITLE
Remove "Global" from the heading on each rst file

### PR DESCRIPTION
**Pull request by Esteban82**
I deleted the word "Global" in the index, now I am deleting on each rst file

### CODE REVIEW

File: docs/earth-age.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-EarthByte Global Earth Seafloor Crustal Age
+EarthByte Earth Seafloor Crustal Age
 -------------------------------------------
```
Comment (seisman) [inline docs/earth-age.rst:2]: Please also make the `-` shorter.
```suggestion
EarthByte Earth Seafloor Crustal Age
------------------------------------
```

Comment (Esteban82) [reply docs/earth-age.rst]: Done.

**Review by seisman — APPROVED** [review]

**Comment by yvonnefroehlich** [comment]
@seisman should this also be updated on the PyGMT side:
```
Admin@A181941 /cygdrive/c/users/Admin/C2/EigeneDokumente/Studium/Promotion/01_GitHub/github_pygmt/pygmt/pygmt/datasets
$ grep "Global" *
earth_free_air_anomaly.py:Function to download the IGPP Global Earth Free-Air Anomaly dataset from the
earth_free_air_anomaly.py:    Load the IGPP Global Earth Free-Air Anomaly dataset in various resolutions.
earth_free_air_anomaly.py:       IGPP Global Earth Free-Air Anomaly dataset.
earth_geoid.py:Function to download the EGM2008 Global Earth Geoid dataset from the GMT data
earth_geoid.py:    Load the EGM2008 Global Earth Geoid dataset in various resolutions.
earth_geoid.py:       EGM2008 Global Earth Geoid dataset.
earth_magnetic_anomaly.py:       * - Global Earth Magnetic Anomaly Model (EMAG2)
earth_magnetic_anomaly.py:        - ``"emag2"``: EMAG2 Global Earth Magnetic Anomaly Model [Default
earth_mask.py:Function to download the GSHHG Global Earth Mask dataset from the GMT data
earth_mask.py:    Load the GSHHG Global Earth Mask dataset in various resolutions.
earth_mask.py:       GSHHG Global Earth Mask dataset.
earth_relief.py:        - ``"igpp"``: IGPP Global Earth Relief [Default option]. See
earth_relief.py:        - ``"synbath"``: IGPP Global Earth Relief dataset that uses
earth_relief.py:        - ``"gebco"``: GEBCO Global Earth Relief with only observed relief and
earth_relief.py:        - ``"gebcosi"``: GEBCO Global Earth Relief that gives sub-ice (si)
earth_vertical_gravity_gradient.py:Function to download the IGPP Global Earth Vertical Gravity Gradient dataset
earth_vertical_gravity_gradient.py:    Load the IGPP Global Earth Vertical Gravity Gradient dataset in various
earth_vertical_gravity_gradient.py:       IGPP Global Earth Vertical Gravity Gradient dataset.
load_remote_dataset.py:        long_name="IGPP Global Earth Free-Air Anomaly",
load_remote_dataset.py:        long_name="EGM2008 Global Earth Geoid",
load_remote_dataset.py:        long_name="IGPP Global Earth Vertical Gravity Gradient",
```

**Comment by seisman** [comment]
I think yes.

**Pull request merged**